### PR TITLE
Replace references to T8 with L8 in the bounds encoding, and provide …

### PR DESCRIPTION
**Pull request by buxtonpaul**
The field T8 is named incorrectly and should be L8 to indicate the field contains bit 8 of the length. Which is used to compute then top field for the capability. 

Resolves issue #(290) https://github.com/riscv/riscv-cheri/issues/290

### CODE REVIEW

File: src/cap-description.adoc [hunk 1]
```diff
@@ -456,15 +456,18 @@ alignment boundaries.
 
 The EF bit selects between two cases:
 
-1. EF = 1: The exponent is 0 for regions less than 2^MW-2^ bytes long
+1. EF = 1: The exponent is 0 for regions less than 2^MW-2^ bytes long. L~8~ is 
+used to encode the MSB of the lenght and is added to B along with T[MW-3:0] to
```
Comment (tariqkurd-repo) [inline src/cap-description.adoc:460]:
```suggestion
used to encode the MSB of the length and is added to B along with T[MW-3:0] to
```

**Comment by buxtonpaul** [comment]
I have re-pushed to the branch fixing both the length typo as well as any line ending issues(I hope)

**Comment by buxtonpaul** [comment]
It was an amend commit which replaced the original

**Review by andresag01 — APPROVED** [review]

**Comment by PRugg-Cap** [comment]
Yep, looks like the right fix

**Pull request merged**